tasks/pull-disk-images/pull-disk-images.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: pull-disk-images
+  labels:
+    app.kubernetes.io/version: "0.1.0"
```

Comment:
shouldn't we go straight to `1.0.0`?

Reply:
Every time we introduce a new task we start with `0.1.0`. Many of our tasks don't even have a 1.0.0 yet

Reply:
oki doki!